pom.xml

```diff
@@ -50,6 +50,10 @@
       <artifactId>jsoup</artifactId>
       <version>1.11.2</version>
     </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-thymeleaf</artifactId>
```

Comment:
if we  decide to use thymeleaf as default rendering engine this can be as "provided"